src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -258,6 +258,10 @@ public ElementTag createOption(String text, String value, boolean isSelected) {
         }
     }
     
+    public ElementTag createOption(String text, String value) {
+        return new ElementTag(text, "value", value);
+    }
+    
```

Comment:
Should this be monopolising the `createOption(String, String)` signature in `PageData`?
Same goes for the `createOption(String, String, boolean)`.